docker/docker_update.sh

```diff
@@ -45,9 +45,9 @@ if [ "$ARCH" = 'arm64' ]; then
   fi
 
   echo "$LOG_PREFIX Initializing images in the background. Please be patient if compiling from source..."
-  docker compose up -d --build
+  docker compose up --build
 else
-  sudo docker compose up -d --build
+  sudo docker compose up --build
```

Comment:
These should still be in the background IMO. After that's fixed feel free to merge.

Reply:
To me it makes much more sense to have it in the foreground, so you can directly see the logs and also stop/restart it.